control-plane-operator/controllers/hostedcontrolplane/autoscaler/reconcile.go

```diff
@@ -284,3 +289,61 @@ func ReconcileAutoscaler(ctx context.Context, c client.Client, hcp *hyperv1.Host
 
 	return nil
 }
+
+const BalancingIgnoreLabelArg = "--balancing-ignore-label"
+
+// The following values are for cloud providers which have not yet created specific nodegroupset processors.
+// These values should be removed and replaced in the event that one of the cloud providers creates a nodegroupset processor.
+
```

Comment:
>// The following values are for cloud providers which have not yet created specific nodegroupset processors.
// These values should be removed and replaced in the event that one of the cloud providers creates a nodegroupset processor.

This comment is misleading. IN our case we use the cluster-api provider and we just decided to drop this from core https://github.com/kubernetes/autoscaler/pull/5375, so there's no intention to cover this by any nodegroupset processor.

Let's drop the comment if that make sense to you as well.